llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -2725,6 +2729,31 @@ void VPExpressionRecipe::print(raw_ostream &O, const Twine &Indent,
     O << ")";
     break;
   }
+  case ExpressionTypes::ExtNegatedMulAccReduction: {
```

Comment:
Is there a way to commonise this with the ExtMulAccReduction case if the only difference is a negate?